### PR TITLE
Troubleshoot install

### PR DESCRIPTION
**Pull request by rosteen**
The current `numpy` max pin is causing install problems, I made it less restrictive and am troubleshooting test/notebook failures with the newer versions of packages.

### CODE REVIEW

**Review by larrybradley — COMMENTED** [review]
Is the change to`wfc3-basic-uvis-fullframe-hstaxe-cookbook.ipynb` intentional?

File: cookbooks/WFC3/UVIS_fullframe_basic_extraction/wfc3-basic-uvis-fullframe-hstaxe-cookbook.ipynb [hunk 1]
```diff
@@ -610,6 +610,8 @@
    "source": [
     "os.chdir(cwd)\n",
     "\n",
+    "print(cwd)\n",
+    "\n",
```
Comment (rosteen) [inline cookbooks/WFC3/UVIS_fullframe_basic_extraction/wfc3-basic-uvis-fullframe-hstaxe-cookbook.ipynb:614]:
```suggestion
```

File: cookbooks/WFC3/UVIS_fullframe_basic_extraction/wfc3-basic-uvis-fullframe-hstaxe-cookbook.ipynb [hunk 2]
```diff
@@ -623,6 +625,8 @@
     "gsrc = 'grism_ims/*flc.fits'\n",
     "csrc = 'WFC3_UVIS_conf/*'\n",
     "\n",
+    "print(csrc)\n",
+    "\n",
```
Comment (rosteen) [inline cookbooks/WFC3/UVIS_fullframe_basic_extraction/wfc3-basic-uvis-fullframe-hstaxe-cookbook.ipynb:629]:
```suggestion
```

**Review by larrybradley — APPROVED** [review]
LGTM, thanks!

**Comment by rosteen** [comment]
> Is the change to`wfc3-basic-uvis-fullframe-hstaxe-cookbook.ipynb` intentional?

The change to the file names (version numbers) was, I just deleted the prints which weren't intentionally included.

**Pull request merged**